runtests.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+import unittest
+from tests import test_m2m
```

Comment:
Should add a blank line separating standard library imports from local project imports.